### DISCUSSION TITLE
Waterfurnace not updating any registers

### DISCUSSION THREAD

**Discussion by phillipredshirt** [ccutrer/waterfurnace_aurora#55] [Q&A]
I've got a new installation(series 5) and did not get the AWL add on knowing that this software was available. When I run 'aurora_fetch /dev/ttyUSB0 valid --yaml > myheatpump.yml' most of the entries are either 0 or 65535. I've been running ' aurora_monitor /dev/ttyUSB0 --diff-only' and I'm getting nothing after an hour of the heat pump turning on and off. 

I definitely have the aurora base control board, the aurora advanced control board and sensors that aurora uses, is it possible it could have been installed in some sort of analog mode where it doesn't even use the digital coms?

I am assuming that I would get nothing or errors from aurora_fetch if I wasn't connected to the unit properly. Maybe that assumption is faulty.  Any ideas?
[myheatpump.zip](https://github.com/ccutrer/waterfurnace_aurora/files/13402383/myheatpump.zip)

**Comment by phillipredshirt** [comment]
Does this software require the Aurora Web Link hardware? I have the AID tool and that works fine, but I am unable to get any real data using aurora_monitor and aurora_fetch.  How can I validate my hardware setup to make sure that's not the issue?

**Comment by ccutrer** [comment]
No, the AWL is not required. I've had mine unplugged for multiple years, because it died and I didn't care.

If it was not able to communicate with the ABC at all, yes, you would get a timeout error. Having a bunch of 0s or 65535 in unknown and unused registers (depending on your configuration) is not unusual. I'll try to find some time to look at your dump to see if there's something the software is not recognizing correctly.

**Comment by phillipredshirt** [comment]
That’s really nice of you, don’t bother unless you’re bored. I’ve double checked the hardware and that looks right. I may just wipe the pi, start over and see if I get the same result.  Knowing I don’t need AWL helps a lot. Thank you.

**Comment by 0n3man** [comment]
Did you make sure your thermostat and WF main board are running a 3.X version of software?  When I started using this code I was getting incorrect information.  I remember getting 65535 values.  My thermostat was running a 1.x version of software.  I had the company that installed the unit come out and upgrade the software in both the WF unit and the thermostat and then all was good.

I added this issue when I was starting out: https://github.com/ccutrer/waterfurnace_aurora/issues/59
